### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by tanjieling0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+
+## User-Freindly Commands
+NetConnect uses simple CLI commands to make it easy for you to manage your contacts.
+
+## Error Handling
+With a robust error handling system, clear error and resolution messages will guide you through the process of 
+managing your contacts seamlessly.
+
+## Getting Started
+To begin your journey with NetConnect, simply download the latest release from the [here]() and refer to the [User Guide](docs/UserGuide.md)
```
Comment (starsia) [inline README.md]:
```suggestion
To begin your journey with NetConnect, simply download the latest release from [here]() and refer to the [User Guide](docs/UserGuide.md)
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
+    * Delete or Edit a profile: Remove a profile from the list or Edit their information via their ID.
+
+2. Category Management
+    * Roles: Categorize your profiles into *Employees, Clients, or Suppliers*.
+
+3. Search
+    * Search by name: Find a profile by their name.
+    * Search by contact: Find a profile by their contact number.
+
+## User-Freindly Commands
```
Comment (starsia) [inline README.md]:
```suggestion
## User-Friendly Commands
```

**Review by starsia — COMMENTED** [review]
Minor typos, otherwise LGTM 🤩

**Comment by tanjieling0** [comment]
fixes #10

File: README.md [hunk 3]
```diff
@@ -1,14 +1,35 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```
Comment (shaunlxw) [inline README.md:1]: need to change this by [generating our own workflow badge](https://dev.to/azure/github-how-to-display-the-status-badge-for-a-github-action-5449). not sure if it's only for Java CI only or for all workflow though..

File: README.md [hunk 4]
```diff
@@ -1,14 +1,35 @@
...
+
+## Error Handling
+With a robust error handling system, clear error and resolution messages will guide you through the process of 
+managing your contacts seamlessly.
+
+## Getting Started
+To begin your journey with NetConnect, simply download the latest release from [here]() and refer to the [User Guide](docs/UserGuide.md)
+for detailed guidance in using the application.
+
+For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
```
Comment (shaunlxw) [inline README.md:34]: i think we need to change the description and link of the documentation to ours

File: docs/images/Ui.png
Comment (shaunlxw) [inline docs/images/Ui.png:1]: our group needs to decide whether to use the term "supplier" or "partner"

**Review by shaunlxw — COMMENTED** [review]
LGTM less the minor changes! we are required to acknowledge the original source of the code (https://se-education.org). i'm not sure if the last line in README.md is sufficient as acknowledgement, or if it'll even be there after we change it to fit our project.

**Pull request merged**